client/my-sites/customer-home/cards/constants.js

```diff
@@ -0,0 +1,21 @@
+export const ACTION_QUICK_LINKS = 'home-action-quick-links';
+export const ACTION_WP_FOR_TEAMS_QUICK_LINKS = 'home-action-wp-for-teams-quick-links';
+export const EDUCATION_FREE_PHOTO_LIBRARY = 'home-education-free-photo-library';
+export const EDUCATION_GUTENBERG = 'home-education-gutenberg';
+export const FEATURE_GO_MOBILE = 'home-feature-go-mobile';
+export const FEATURE_QUICK_START = 'home-feature-quick-start';
+export const FEATURE_STATS = 'home-feature-stats';
+export const FEATURE_SUPPORT = 'home-feature-support';
+export const NOTICE_CELEBRATE_SITE_CREATION = 'home-notice-celebrate-site-creation';
+export const NOTICE_CELEBRATE_SITE_LAUNCH = 'home-notice-celebrate-site-launch';
+export const NOTICE_CELEBRATE_SITE_MIGRATION = 'home-notice-celebrate-site-migration';
+export const NOTICE_CELEBRATE_SITE_SETUP_COMPLETE = 'home-notice-celebrate-site-setup-complete';
+export const SECTION_LEARN_GROW = 'home-section-learn-grow';
+export const SECTION_MANAGE_SITE = 'home-section-manage-site';
+export const TASK_CONNECT_ACCOUNTS = 'home-task-connect-accounts';
+export const TASK_FIND_DOMAIN = 'home-task-find-domain';
+export const TASK_GO_MOBILE_ANDROID = 'home-task-go-mobile-android';
+export const TASK_GO_MOBILE_IOS = 'home-task-go-mobile-ios';
+export const TASK_SITE_SETUP_CHECKLIST = 'home-task-site-setup-checklist';
+export const TASK_WEBINARS = 'home-task-webinars';
+export const TASK_EDITOR_DEPRECATION = 'home-task-editor-deprecation';
```

Comment:
Non-blocking question: @Automattic/team-calypso are there any recommendations here for keeping bundle size reasonable if this file grows in size?